### ISSUE TITLE
[docs] recommend vscode prettier plugin

### ISSUE THREAD

**Issue by LarsDenBakker** [open-wc/open-wc#918]
I think we should recommend the vscode prettier plugin for vscode

**Comment by LarsDenBakker** [comment]
Not stale

**Comment by davie-robertson** [comment]
I agree - came here trying to figure out why the prettier config doesn't seem to do much in vscode. I'm still none the wiser - even after running ` npm run format:prettier` the code still looks bad.
